tests/test_ac_harness_racing_telemetry.py

```
@@ -0,0 +1,81 @@
+
+from tools.ac_harness import racing_telemetry
+from tools.ac_harness.racing_telemetry import (
+    CSV_HEADER,
+    csv_row,
+    parse_graphics,
+    parse_physics,
+    record,
+)
+
+
+def _phys_buf(**kw) -> bytes:
```

Ruff (0.15.17)

[warning] Missing type annotation for `**kw`

(ANN003)